nemo_skills/pipeline/openrlhf/datasets.py

```diff
@@ -0,0 +1,57 @@
+
```

Comment:
Please add both nvidia and original license as well as a reference to where this code originally comes from